### ISSUE TITLE
Quasar Framework preset

### ISSUE THREAD

**Issue by IlCallo** [laravel-frontend-presets/internals#6]
[Quasar framework](http://quasar-framework.org/), based on Vue, is gaining momentum and will probably release it's v1.0 in some months.
Would be nice to cooperate with it's maintainer to create a preset for it, given that it doesn't seem too difficult, being mostly based on Vue which is more or less already supported.

Something has already been done:
- [Laravel-Quasar in UMD mode](https://github.com/neatpro/Laravel-Quasar)
- [Laravel5.4 and Quasar](http://blog.aprilpnguyen.com/2017/08/how-to-set-up-quasar-v014-on-laravelvue.html)

Any thoughs?

**Comment by m1guelpf** [comment]
ping @rstoenescu @claustres @zuck Would any of you be interested in mantaining a Laravel preset for Quasar?

**Comment by OwenMelbz** [comment]
@IlCallo There's no reason that it cannot be added here :)

It is probably worth saying that as a whole, all the packages on here - are people who have built something, and wanted a central place to share them.

Rather than asking us to build them.

e.g the guys who create the Tailwind or Bulma preset, might never have looked at Quasar, however if there was somebody who created one, they are more than welcome to share on here :)

**Comment by claustres** [comment]
Sorry but not enough time to manage my current own projects :-(

**Comment by IlCallo** [comment]
Yeah, we (my company) is starting to study Quasar and how to integrate it with Laravel, that's why I was asking if there was some interest in this topic and/or if someone already used Quasar with Laravel and wants to share it's experience

**Comment by rstoenescu** [comment]
Hi, Please join our [Discord](https://discord.gg/5TDhbDg) server. We have a #laravel channel there maintained by CampusCloud who is our Laravel master. We can plan what needs to be done there if it's ok with you.

**Comment by SirAuron** [comment]
Hi all.
I'm **IlCallo** colleague. We've discussed a bit and we've come up that I'll be the one to follow this project.
Today and tomorrow I've a bit of work to do, so I'll not be able to join the channel right away. I think I'll do that on monday.
